e2e/validatingGraphs.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+
+const GRAPHS = [
+  // SDP Pipelines
+  "https://raw.githubusercontent.com/ICRAR/EAGLE-graph-repo/refs/heads/master/SDP%20Pipelines/cont_img_mvp.graph",
+
+  // examples
+  "https://raw.githubusercontent.com/ICRAR/EAGLE-graph-repo/refs/heads/master/examples/HelloWorld-simple.graph",
+
+  // graph_patterns
+  "https://raw.githubusercontent.com/ICRAR/EAGLE-graph-repo/refs/heads/master/graph_patterns/LoopWithBranch.graph"
+]
+
+test('Validating Graphs', async ({ page }) => {
+  page.setViewportSize({width:1920,height:1080})
```

Comment:
playing around with this branch i noticed we dont seem to need this line. its also used in some of the other tests but this is already set as default so we dont need to re-set it. Additionally running this locally in headed mode(showing browser) i noticed the bottom of eagle is cut off when this line is enabled, after disabling it, it looks normal. maybe should remove these.

Reply:
OK, I'll remove it. Thanks.